projects/packages/jitm/src/js/jetpack-jitm.js

```diff
@@ -206,8 +207,16 @@ jQuery( document ).ready( function ( $ ) {
 				},
 			} ).done( function () {
 				$( '#jitm-banner__activate a' ).text( window.jitm_config.activated_module_text );
-				$( '#jitm-banner__activate a' ).attr( 'disabled', true );
+				
+				// Present a link to module settings page if a settings link is set in the JITM envelope.
+				if ( $activate_button.data( 'settings_link' ) ) {
+					$( '#jitm-banner__activate a' ).text( window.jitm_config.settings_module_text );
+					$( '#jitm-banner__activate a' ).attr( 'href', window.jitm_config.jetpack_admin_url + $activate_button.data( 'redirect' ) );
```

Comment:
Nice improvement over the redirect 👍  

What would you think about passing the full settings URL for the activated module in the JITM response and adding a settings button in the JITM template? 
Then, when you get to this step, you could just hide the activation button and show the settings button.

This allows us to avoid splitting up the URL into two parts, one that comes from the plugin and one that comes from the JITM response and having to re-assemble here. It will likely be less fragile to have the entire settings URL coming from a single source.

Reply:
Yeah, I think I'll do this - have another button drawn if it has the data in the envelope and then show hide it with JS